internal/cmd/compilecheck/go.mod

```diff
@@ -12,7 +12,6 @@ require go.mongodb.org/mongo-driver v1.11.7
 require (
 	github.com/golang/snappy v0.0.1 // indirect
 	github.com/klauspost/compress v1.13.6 // indirect
-	github.com/montanaflynn/stats v0.0.0-20171201202039-1bf9dbcd8cbe // indirect
```

Comment:
That's a good question. This was done automatically, I'm assuming it's because internal/benchmark isn't needed for our use case of the dependency:

> If needed, it adds require directives to your go.mod file for modules needed to build packages named on the command line. A require directive tracks the minimum version of a module that your module depends on: https://go.dev/doc/modules/managing-dependencies#adding_dependency